src/features/catalog/CatalogPage.tsx

```diff
@@ -1,20 +1,31 @@
 import Box from '@mui/material/Box';
 import ProductList from './components/ProductList/ProductList';
+import CategoryList from './components/CategoryList/CategoryList';
+import { CategoryProvider } from '../../providers/CategoriesProvider';
+import CustomBreadcrumbs from './components/Breadcrumbs/Breadcrumbs';
 
 export default function CatalogPage() {
   return (
-    <Box
-      component="div"
-      sx={{
-        maxWidth: { lg: '1108px', md: '850px', xs: '90%' },
-        display: 'flex',
-        flexDirection: 'column',
-        gap: 2,
-        margin: 'auto',
-        py: { lg: 8, md: 6, xs: 4 },
-      }}
-    >
-      <ProductList />
-    </Box>
+    <CategoryProvider>
+      <Box
+        component="div"
+        sx={{
+          maxWidth: { lg: '1108px', md: '850px', xs: '90%' },
+          gap: 2,
+          margin: 'auto',
+          py: { lg: 8, md: 6, xs: 4 },
+        }}
+      >
+        <Box width="100%" display="flex" gap={2}>
+          <Box component="div" minWidth="30%" height="fit-content">
+            <CategoryList />
+          </Box>
+          <Box maxWidth="calc(70% - 16px)" display="flex" flexDirection="column" gap={1}>
+            <CustomBreadcrumbs />
+            <ProductList />
+          </Box>
```

Comment:
Can we use grid with collumns?

```suggestion
        <Grid container spacing={3}>
          <Grid size={3}>
            <CategoryList />
          </Grid>
          <Grid size={9} display={'grid'} gap={2}>
            <CustomBreadcrumbs />
            <ProductList />
          </Grid>
        </Grid>
```
Preview
<img width="1214" alt="Screenshot 2025-05-28 at 16 10 39" src="https://github.com/user-attachments/assets/8dbdcb9c-943e-4167-ae3b-38d3b73c9ff1" />
:

Reply:
in this case we need to add 100% width property to parent box where we was set maxWidth

Reply:
in this case we need to add 100% width property to parent box where we was set maxWidth

Reply:
Thank you for suggestions, improved design.